docs/message.md

```diff
@@ -0,0 +1,65 @@
+# Message API
+
+The Message API aims to cover the Messages part of the [REST API documentation](https://developers.symphony.com/restapi/reference#messages-v4).
+More precisely:
+* [Get a message](https://developers.symphony.com/restapi/reference#get-message-v1)
+* [Get messages](https://developers.symphony.com/restapi/reference#messages-v4)
+* [Get message IDs by timestamp](https://developers.symphony.com/restapi/reference#get-message-ids-by-timestamp)
+* [Send message](https://developers.symphony.com/restapi/reference#create-message-v4)
+* [Import messages](https://developers.symphony.com/restapi/reference#import-message-v4)
+* [Get attachment](https://developers.symphony.com/restapi/reference#attachment)
+* [List attachments](https://developers.symphony.com/restapi/reference#list-attachments)
+* [Get allowed attachment types](https://developers.symphony.com/restapi/reference#attachment-types)
+* [Suppress message](https://developers.symphony.com/restapi/reference#suppress-message)
+* [Get message status](https://developers.symphony.com/restapi/reference#message-status)
+* [Get message receipts](https://developers.symphony.com/restapi/reference#list-message-receipts)
+* [Get message relationships](https://developers.symphony.com/restapi/reference#message-metadata-relationship)
+
+## How to use
+The central component for the Message API is the `MessageService`.
+It exposes all the services mentioned above and is accessible from the `SymphonyBdk` object by calling the `messages()` method.
+For instance:
+
+```java
+import com.symphony.bdk.core.MessageService;
+
+public class Example {
+  public static final String STREAM_ID = "gXFV8vN37dNqjojYS_y2wX___o2KxfmUdA";
+
+  public static void main(String[] args) throws Exception {
+    // Create BDK entry point
+    final SymphonyBdk bdk = new SymphonyBdk(loadFromSymphonyDir("config.yaml"));
+    // Get the MessageService object
+    final MessageService messageService = bdk.message();
+
+    //send a regular message
+    final V4Message regularMessage = messageService.send(STREAM_ID, "<messageML>Hello, World!</messageML>");
+    System.out.println("Message sent, id: " + regularMessage.getMessageId());
+  }
+}
+```
```

Comment:
Will reference the example class MessageExampleMain when https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/pull/200 is merged